lib/jsdom/browser/Window.js

```diff
@@ -119,21 +119,19 @@ function Window(options) {
   Object.defineProperty(idlUtils.implForWrapper(this), idlUtils.wrapperSymbol, { get: () => this._globalProxy });
 
   // List options explicitly to be clear which are passed through
-  this._document = Document.create(window, [], {
-    options: {
-      parsingMode: options.parsingMode,
-      contentType: options.contentType,
-      encoding: options.encoding,
-      cookieJar: options.cookieJar,
-      url: options.url,
-      lastModified: options.lastModified,
-      referrer: options.referrer,
-      concurrentNodeIterators: options.concurrentNodeIterators,
-      parseOptions: options.parseOptions,
-      defaultView: this._globalProxy,
-      global: this
-    }
-  });
+  this._document = documents.createWrapper(window, {
+    parsingMode: options.parsingMode,
+    contentType: options.contentType,
+    encoding: options.encoding,
+    cookieJar: options.cookieJar,
+    url: options.url,
+    lastModified: options.lastModified,
+    referrer: options.referrer,
+    concurrentNodeIterators: options.concurrentNodeIterators,
+    parseOptions: options.parseOptions,
+    defaultView: this._globalProxy,
+    global: this
+  }, { alwaysUseDocumentClass: true });
```

Comment:
Windows with XML documents in them create `XMLDocument` instances.

```suggestion
  });
```

Reply:
That is incorrect per spec.

Reply:
Well, this does result in `true` in all browsers:

```xml
<doc xmlns:html="http://www.w3.org/1999/xhtml">
<html:script><![CDATA[
document.documentElement.append(`${document instanceof XMLDocument}`);
]]></html:script>
</doc>
```

This is necessary for webcompat.

Reply:
We won't implement that in jsdom without a spec.

Reply:
Well, <https://github.com/whatwg/html/pull/5104> does fix that.

Reply:
If/when that moves forward we can consider it in jsdom.